src/noos_tf/cli.py

```diff
@@ -30,8 +30,8 @@ def run(ctx, message="", workspace="", organisation=None, token=None):
 
 
 ns = Collection()
-ns.add_task(update)
```

Comment:
⚠️ 

Current issue reported within the `invoke` project
https://github.com/pyinvoke/invoke/issues/946

What about adding a `# type: ignore` annotation as well as a `TODO` with a reference to the current opened Github issue?

Reply:
sound good to me